src/main.rs

```diff
@@ -30,18 +38,47 @@ fn main() -> anyhow::Result<()> {
         }
     }
 
+    let start = OffsetDateTime::now_utc();
+
     let lockfile = Lockfile::load(LOCKFILE).context("failed to load lockfile")?;
     let cargo_toml = load_toml(CARGO_TOML).context("failed to load Cargo.toml")?;
     let packages = discover_packages(&cargo_toml).context("failed to discover packages")?;
     let dependency_tree = lockfile.dependency_tree().context("failed to generate dependency tree")?;
     let database = Database::fetch().context("failed to fetch advisory-db")?;
     let vulnerabilities = database.vulnerabilities(&lockfile);
 
+    let end = OffsetDateTime::now_utc();
+
     print_vulnerabilities(&vulnerabilities);
 
+
     let report = report::Report {
         version: REPORT_VERSION.to_string(),
         vulnerabilities: report_vulnerabilities(&vulnerabilities),
+        scan: report::Scan {
+            analyzer: report::Analyzer {
+                id: SCANNER_ID.to_string(),
+                name: SCANNER_NAME.to_string(),
+                url: None,
+                vendor: report::Vendor {
+                    name: SCANNER_VENDOR.to_string(),
+                },
+                version: env!("CARGO_PKG_VERSION").to_string(),
+            },
+            scanner: report::Scanner {
+                id: SCANNER_ID.to_string(),
+                name: SCANNER_NAME.to_string(),
+                url: None,
```

Comment:
Would you mind adding `"https://crates.io/crates/cargo-audit"` as the `url` of the `scanner`.